builtin/credential/okta/path_config.go

```diff
@@ -118,6 +129,31 @@ func (b *backend) pathConfigWrite(
 		cfg.BaseURL = d.Get("base_url").(string)
 	}
 
+	var ttl time.Duration
+	ttlRaw, ok := d.GetOk("ttl")
```

Comment:
Once you switch to TypeDurationSecond most of this goes away -- you are guaranteed to get a valid `int` out of a `Get` call. So a zero value can just mean "not set" and can be used directly.